client/transactions/list/index.tsx

```diff
@@ -612,7 +619,7 @@ export const TransactionsList = (
 					transactionsSummary.count as number,
```

Comment:
Do we need to format this here as well? If yes, I'd recommend storing the formatted value as a variable above and then using that directly.

Reply:
`transactionsSummary.count` is passed as an argument to the `_n` function. And the function [expects an integar](https://developer.wordpress.org/reference/functions/_n/#parameters) there. So I don't think we need to apply formatting there.